### PR TITLE
PLAT-128468: Add UI tests and unit tests for FanSpeedControl

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for FanSpeedControl
 - Added unit tests for FanSpeedControl

### Links
[//]: # (Related issues, references)
PLAT-128468

### Comments

Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

File: tests/ui/specs/FanSpeedControl/FanSpeedControl-specs.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		it('should select all arcs up to the clicked arc', function () {
+			fanSpeedControl.clickablePath(3).click();
+			expect(fanSpeedControl.coloredPath(1).getCSSProperty('stroke').value).to.equal('rgb(68,68,68)');
+			expect(fanSpeedControl.coloredPath(2).getCSSProperty('stroke').value).to.equal('rgb(68,68,68)');
+			expect(fanSpeedControl.coloredPath(3).getCSSProperty('stroke').value).to.equal('rgb(68,68,68)');
+			// next one is unselected
+			expect(fanSpeedControl.coloredPath(4).getCSSProperty('stroke').value).to.equal('rgb(238,238,238)');
+		});
+
+		it('should display `fan` icon', function () {
```
Comment (daniel-stoian-lgp) [inline tests/ui/specs/FanSpeedControl/FanSpeedControl-specs.js:24]: can you also have a FanSpeed component in the view with a custom icon and do 1 test to check the icon?

Comment (adrian-cocoara-lgp) [reply tests/ui/specs/FanSpeedControl/FanSpeedControl-specs.js:24]: sure

**Review by daniel-stoian-lgp — APPROVED** [review]

File: tests/ui/specs/FanSpeedControl/FanSpeedControl-specs.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+describe('FanSpeedControl', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	describe('default', function () {
+		const fanSpeedControl = Page.components.fanSpeedControlDefault;
+
+		it('should have the first arc selected by default', function () {
+			expect(fanSpeedControl.coloredPath(1).getCSSProperty('stroke').value).to.equal('rgb(68,68,68)');
```
Comment (jeonghee27) [inline tests/ui/specs/FanSpeedControl/FanSpeedControl-specs.js:12]: In fact, there is a screenshot test for this part, so a function-oriented test should be conducted in the ui test.
Anyway, it's an implemented part. So we just keep it.

**Review by jeonghee27 — APPROVED** [review]
LGTM
Sorry to late reveiw

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**